test/metabase/upload_test.clj

```diff
@@ -1068,6 +1043,31 @@
          ::upload/append (into initial added)
          ::upload/replace added)))
 
+(defn update-csv-with-defaults!
```

Comment:
I moved it down here because `can-update-test` is the only test in this namespace that uses it, so it's nice to have it close by